server/routerlicious/packages/lambdas/package.json

```diff
@@ -61,6 +61,7 @@
     "@fluidframework/server-services-client": "^0.1038.3000",
     "@fluidframework/server-services-core": "^0.1038.3000",
     "@fluidframework/server-services-telemetry": "^0.1038.3000",
+    "@fluidframework/server-services-utils": "^0.1038.3000",
```

Comment:
We cannot import service-utils in the lambdas package. Utils are not isomorphic, but lambdas are.